unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentView.cs

```diff
@@ -107,7 +113,21 @@ public override void OnLoseFocus()
         public void OnSlotClick()
         {
             isSelected = !isSelected;
-            selectedImage.enabled = isSelected;
+
+            if (isSelected)
+            {
+                selectedImage.enabled = true;
+                selectedImage.transform.localScale = new Vector3(0, 0, 0);
+                selectedImage.transform.DOScale(1, ANIMATION_TIME).SetEase(Ease.OutBack);
+            }
+            else
+            {
+                selectedImage.transform.DOScale(0, ANIMATION_TIME).SetEase(Ease.OutBack).OnComplete(() =>
+                {
+                    selectedImage.enabled = false;
+                    selectedImage.transform.localScale = new Vector3(1,1,1);
+                });
+            }
```

Comment:
Can we extract all the animations logic into separated methods? In this way we make the code cleaner and understandable. Wdyt?